extensions/mathsys.py

```
@@ -26,7 +26,7 @@ async def init(self, socket: æ.WebSocket) -> namespace: return self
     def process(self, code: str) -> bool:
         self.code = code
         try:
-            mathsys.compile(self.code)
+            self.compile()
             return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)